logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultChronoZonedDateTimeAssertions.kt

```diff
@@ -44,4 +47,86 @@ class DefaultChronoZonedDateTimeAssertions : ChronoZonedDateTimeAssertions {
     ): Assertion = container.createDescriptiveAssertion(IS_EQUAL_TO, expected) {
         it.isEqual(expected)
     }
+
+    override fun <T : ChronoZonedDateTime<out ChronoLocalDate>> isBefore(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion {
+        return container.isBefore(parseZonedDateTime(expected))
+    }
```

Comment:
detail but would be nice if you could change (and the following) to a one-liner
```suggestion
    ): Assertion = container.isBefore(parseZonedDateTime(expected))
```